swagger.yaml

```diff
@@ -5469,6 +5469,7 @@ definitions:
       - kernel_url
       - rootfs_url
       - initrd_url
+      - host_fs_mount_dir
```

Comment:
Maybe `host_filesystem_dir` or `host_filesystem_mount` if we want to fully avoid abbreviations?

@avishayt thoughts? Naming is hard :confounded: